vcpkg/get_started/get-started-packaging.md

```diff
@@ -0,0 +1,210 @@
+
+## Prerequisites
+
+- A terminal
+- A C++ compiler
+- [CMake](https://cmake.org/download/)
+- [Git](https://git-scm.com/downloads)
+
+::: zone pivot="os-windows"
+
+> [!NOTE]
+> For Windows users, Visual Studio's MSVC (Microsoft Visual C++ Compiler) is the required compiler for C++ development.
+
+::: zone-end
+
+## 1 - Set up vcpkg
+
+[!INCLUDE [setup-vcpkg](includes/setup-vcpkg.md)]
+
+## 2 - Configure the `VCPKG_ROOT` environment variable
+
+To set the `VCPKG_ROOT` environment variables, run the following commands:
+
+::: zone pivot="os-linux,os-macos"
+
+```bash
+export VCPKG_ROOT=/path/to/vcpkg
+export PATH=$VCPKG_ROOT:$PATH
+```
+
+> [!NOTE]
+> Setting the `VCPKG_ROOT` environment variable using the `export` command only affects the current shell session. To make this change permanent across sessions, you'll need to add the `export` command to your shell's profile script (e.g., `~/.bashrc` or `~/.zshrc`).
+
+::: zone-end
+
+::: zone pivot="os-windows"
+
+```console
+set VCPKG_ROOT="C:\path\to\vcpkg"
+set PATH=%VCPKG_ROOT%;%PATH%
+```
+
+> [!NOTE]
+> Setting the `VCPKG_ROOT` environment variable using the `set` command only affects the current shell session. To make this change permanent across sessions, you can use the `setx` command and restart the shell session.
+
+::: zone-end
+
+Setting `VCPKG_ROOT` tells vcpkg where your vcpkg instance is located.
+Adding it to `PATH` ensures you can run vcpkg commands directly from the shell.
+
+## 3 - Set up the port files
+
+1. First, create a new folder under the `vcpkg/ports/` directory named `vcpkg-sample-library`
+
+2. Next, create the `vcpkg.json` file within the `vcpkg-sample-library` folder with the following content:
+   
+   ```json
+    {
+    "name": "vcpkg-sample-library",
+    "version": "1.0.3",
+    "homepage": "https://github.com/JavierMatosD/vcpkg-sample-library",
+    "description": "A sample C++ library designed to serve as a foundational example for a tutorial on packaging libraries with vcpkg.",
+    "license": "MIT",
+    "dependencies": [
+        {
+        "name" : "vcpkg-cmake",
+        "host" : true
+        },
+        {
+        "name" : "vcpkg-cmake-config",
+        "host" : true
+        },
+        {
+        "name": "fmt"
+        }
+    ]
+    }
+    ```
+    TODO: provide line-by-line explanation
+    TODO: For more information on `vcpkg.json`, see the following:
+
+3. Now, create the `usage` file within the `vcpkg-sample-library` folder with the following content:
+
+   ```
+   vcpkg-sample-library provides CMake targets:
+   
+   find_package(my_sample_lib CONFIG REQUIRED)
+   target_link_libraries(main PRIVATE my_sample_lib::my_sample_lib)
+   ```
+   Providing usage documentation for ports allows users to easily adopt them in their projects. We highly encourage providing a `usage` file within the port's directory (`ports/<port name>/usage`) that describes the minimal steps necessary to integrate with a build system.
```

Comment:
We should mention that to know what the correct usage is:
* the port author uses whatever is instructed by upstream, or
* the port author goes through the library's build system and finds the exported targets (not always easy).